modules/cluster-logging-manual-rollout-rolling.adoc

```diff
@@ -171,12 +183,14 @@ $ oc exec elasticsearch-cdm-5ceex6ts-1-dcd6c4c7c-jpw6 -c elasticsearch -- es_uti
 
 . After all the deployments for the cluster have been rolled out, re-enable shard balancing:
 +
+[source,terminal]
 ----
 $ oc exec <any_es_pod_in_the_cluster> -c elasticsearch -- es_util --query="_cluster/settings" -XPUT -d '{ "persistent": { "cluster.routing.allocation.enable" : "all" } }'
 ----
 +
 For example:
 +
+[source,terminal]
 ----
 $ oc exec elasticsearch-cdm-5ceex6ts-1-dcd6c4c7c-jpw6 -c elasticsearch -- es_util --query="_cluster/settings" -XPUT -d '{ "persistent": { "cluster.routing.allocation.enable" : "all" } }'
 ----
```

Comment:
line 220:  logging-fluentd ---> collector. Remove the label

oc -n openshift-logging patch daemonset/collector --type=json  -p '[{"op": "remove", "path":"/spec/template/spec/nodeSelector/logging-infra-fluentd"}]'